### DISCUSSION TITLE
Episode availability

### DISCUSSION THREAD

**Discussion by AlexVeeBee** [seerr-team/seerr#280] [Ideas]
# Episode availability
if the episode is available then the block should go green like this
![opera_UahifNeYPQ](https://user-images.githubusercontent.com/75509525/208519112-f7aff29c-8b69-41ea-aaaf-4911e7a6d799.png)
or like this
![opera_OwO9NHDbby](https://user-images.githubusercontent.com/75509525/208518072-20cca2f3-f370-46fa-b0a9-c59a1268fdb2.png)
or like how its displayed normally like for the movies when its requested or available
![image](https://user-images.githubusercontent.com/75509525/208521644-9632e3e1-fff6-4f51-9795-745dd622a5f8.png)

Any questions?

**Comment by fallenbagel** [comment]
Episode availability is in the works upstream

**Reply by AlexVeeBee** [reply]
so are you guys working on it?
